test/node_test_setup.js

```diff
@@ -3,6 +3,15 @@ var chalk = require('chalk');
 var diff = require('deep-object-diff').diff;
 var commander = require('commander');
 
+// TODO remove when node 14 fades out
+// node 14 test polyfill
+if (Number(process.versions.node.split('.')[0]) < 16) {
```

Comment:
can't we just check for the existance of global.AbortController?

Reply:
DONE
Let's merge